tests/page_tests.py

```diff
@@ -95,3 +101,59 @@ def remove_bold_and_italics(text):
             self.fail("The following words were spelled incorrectly in file {}: \n    {}".format(
                 self.page, "\n    ".join(failed_words)
             ))
+
+    def test_GIVEN_a_page_IF_it_contains_urls_WHEN_url_loaded_THEN_response_is_http_ok(self):
+        # Have to open as UTF-8. Opening as ascii causes some encoding errors.
+        try:
+            with codecs.open(self.page, encoding="utf-8") as wiki_file:
+                text = wiki_file.read()
+        except Exception:
+            self.fail("FAILED TO OPEN {}".format(self.page))
```

Comment:
This would give us a bit more information about what failed - otherwise we lose the exception trace:

```suggestion
            self.fail("FAILED TO OPEN {} because {} : {}".format(self.page, e.__class__.__name__, e))
```